.github/workflows/create-charts.sh

```diff
@@ -32,6 +32,12 @@ update_data_services_charts() {
         echo '  - name: mongodb'
         echo '    repository:' $HELM_GCS_REPO
         echo '    version:' $mongodb_version
+        echo '    condition: mongodb.enabled'
+        echo '  - name: postgresql'
```

Comment:
can we keep name as `postgres`?

Reply:
Here, it's chart name which is not something we change. We have already overridden it in values. 

see this: https://github.com/hypertrace/hypertrace/blob/d8b28259dac095ba80d1c74ef2d42b6b4a8b91e0/kubernetes/data-services/postgres/values.yaml#L8